VariablesHomework/VariablesHomework.cs

```diff
@@ -8,50 +8,59 @@ public class Variables
         [TestMethod]
         public void Declaration()
```

Comment:
This block of tests is about Declaration. Don't change the Asserts in this block. Instead, add variable declarations (and initializations) so the Asserts pass.